### PR TITLE
Update the read local config function in config tuner.

### PR DESCRIPTION
**Pull request by Antlera**
### What changes were proposed in this pull request?

Update read local config in config tuner.

### Why are the changes needed?

Config tuner need to read local config to update paral config.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

UT.

### CODE REVIEW

**Review by merlintang — APPROVED** [review]
LGTM

File: dlrover/python/elastic_agent/config/paral_config_tuner.py [hunk 1]
```diff
@@ -33,32 +38,68 @@ def __init__(self):
...
         Updates the parallelism configuration every 30 seconds. This method is
         intended to run on a separate thread started by `self.start`.
         """
         while True:
-            config: ParallelConfig = self._master_client.get_paral_config()
-            with open(self.config_path, "w") as f:
-                f.write(config.to_json())
+            local_config = self._read_paral_config(self.config_path)
+            self._master_client.report_paral_config(local_config)
+            logger.info(f"Reported parallelism config: {local_config}")
```
Comment (workingloong) [inline dlrover/python/elastic_agent/config/paral_config_tuner.py:59]: The log will be very frequent.

File: dlrover/python/elastic_agent/config/paral_config_tuner.py [hunk 2]
```diff
@@ -33,32 +38,68 @@ def __init__(self):
...
             time.sleep(30)
+            config: ParallelConfig = self._master_client.get_paral_config()
+            logger.info(f"Received parallelism config: {config}")
```
Comment (workingloong) [inline dlrover/python/elastic_agent/config/paral_config_tuner.py:62]: The log will be very frequent.

File: dlrover/python/elastic_agent/config/paral_config_tuner.py (hunk 1 above)
Comment (Antlera) [reply dlrover/python/elastic_agent/config/paral_config_tuner.py]: Fixed in the lastest commit. This is a left over of development.

File: dlrover/python/elastic_agent/config/paral_config_tuner.py (hunk 2 above)
Comment (Antlera) [reply dlrover/python/elastic_agent/config/paral_config_tuner.py]: Fixed in the lastest commit. This is a left over of development.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**